source/components/JsxParser.tsx

```diff
@@ -176,10 +177,14 @@ export default class JsxParser extends React.Component<TProps> {
 					.map(item => this.#parseExpression(item, scope))
 					.join('')
 			case 'UnaryExpression':
+				const unaryValue = this.#parseExpression(
+					expression.argument as AcornJSX.Expression,
+					scope,
+				)
```

Comment:
this is needed in order to handle `-Infinity` correctly. This value actually comes through as a unary operator `-` along with the Literal `Infinity` — but by parsing it, we can get it back as `-Infinity` correctly.